dotcom-rendering/src/web/lib/parser/jsonParser.ts

```diff
@@ -0,0 +1,16 @@
+export const safeJsonParse =
```

Comment:
Couldn't see anywhere we were doing generic json parsing. Correct me if I'm wrong though

Reply:
Checked with Olly and this looks ok